starlite/cli.py

```diff
@@ -0,0 +1,326 @@
+from __future__ import annotations
```

Comment:
Please keep consistency with the rest of the code base.

Reply:
Didn't we say we would start to adopt this a while ago? I might be mistaken but I think I remember something along the lines of wanting to use it for new things and upgrading other stuff on an as-needed basis.

Reply:
we discussed migrating everything in one go. For now we should keep everything consistent I think. 

Also - can you increase coverage?